core/src/main/java/com/opentable/logging/JsonLogEncoder.java

```diff
@@ -41,6 +41,7 @@
     private static final AtomicLong LOG_SEQUENCE_NUMBER = new AtomicLong(0);
 
     private final ObjectMapper mapper;
+    private Class<?> httpEventClass = HttpLogFields.class;
```

Comment:
This isn't inherently a http event class, would it be better to call it maybe 'typeToSerialize' or something since it is actually useful for even non-HTTP things?